ros_ign_gazebo_demos/launch/tf_bridge.launch.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import ExecuteProcess
+from launch_ros.actions import Node
+
+def generate_launch_description():
+    pkg_ros_ign_gazebo_demos = get_package_share_directory('ros_ign_gazebo_demos')
+    return LaunchDescription([
+        # Launch gazebo
+        ExecuteProcess(
+            cmd=['ign','gazebo', '-r', os.path.join(pkg_ros_ign_gazebo_demos,
+                    'models', 'double_pendulum_model.sdf')]
+        ),
+        # Launch a bridge to forward tf and joint states to ros2
+        Node(
+            package='ros_ign_bridge',
+            executable='parameter_bridge',
+            arguments=[
+                '/world/default/model/double_pendulum_with_base0/joint_state@sensor_msgs/msg/JointState[ignition.msgs.Model',
```

Comment:
do you mind to remap `joint_state` too ?

Reply:
Done https://github.com/ignitionrobotics/ros_ign/pull/244/commits/e013b3cccf4b1ec08ce31a9fdd24c46f8815f9d2